src/ecdcpipeline/DockerWrapper.groovy

```diff
@@ -0,0 +1,49 @@
+package ecdcpipeline
+
+import ecdcpipeline.DockerOutputParser
+
+/**
+ * Wrapper for Docker commands.
+ */
+class DockerWrapper implements Serializable {
+
+  private def script
+  private def dockerOutputParser
+
+  /**
+   * <p></p>
```

Comment:
should there be something between the <>s? or could that line be deleted?

Reply:
I used to believe this was necessary due to some issue that occurred if we didn't have it, but I don't remember what it was and could not see any difference in the documentation by removing it, so I removed it.